dist/install-elixir.ps1

```diff
@@ -0,0 +1,5 @@
+Write-Output "Installer for Elixir for Windows not available"
```

Comment:
He/she/it who comes after is more than welcome to take a stab at this. For the time being I preferred to be explicit in saying Elixir is not available.